### PR TITLE
wip

### PR DESCRIPTION
**Pull request by ymgyt**

### CODE REVIEW

**Comment by ymgyt** [comment]
self comment

**Comment by howtv-ymgyt** [comment]
comment

**Comment by howtv-ymgyt** [comment]
hello

**Comment by howtv-ymgyt** [comment]
hello

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v1.0.4
\ No newline at end of file
+v1.0.5
```
Comment (howtv-ymgyt) [inline VERSION]: nice bump

Comment (ymgyt) [reply VERSION]: thanks !

Comment (ymgyt) [reply VERSION]: self comment

Comment (ymgyt) [reply VERSION]: never notifify

**Pull request closed without merging**